keyboards/blackplum/config.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0099
+#define PRODUCT_ID      0x6800
```

Comment:
Is this an unused ID?

Reply:
VENDOR_ID  0x0099 according to https://usb-ids.gowdy.us/read/UD?restrict=0 is unused ID, should I leave to default 0xFEED ?